Mathlib/Data/ZMod/Zero.lean

```diff
@@ -0,0 +1,15 @@
+/-
+Copyright (c) 2025 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+
+import Mathlib.Algebra.Equiv.TransferInstance
+import Mathlib.Data.ZMod.Defs
+import Mathlib.Algebra.Order.Ring.Int
+
+/-! `ZMod 0` is a domain -/
+
+/-- `ZMod 0` is a domain -/
+instance : IsDomain (ZMod 0) :=
```

Comment:
This can be proved as
```lean
instance : IsDomain (ZMod 0) :=
  inferInstanceAs (IsDomain ℤ)
```
And there are surely files where it can go, without changing the imports.